lib/middleware/websocket/interactors/build_payload.rb

```diff
@@ -1,12 +1,16 @@
 module Websocket
   module Interactor
     class BuildPayload
-      def call(clients: clients)
+      def race(clients:)
```

Comment:
Generally a service object will always have a `call` method. Why are we changing this?

Reply:
@jgayfer 
My first idea was to implement `BuildPayload` as follows:
```
      def call(clients:, type:)
        case type
        when 'race'
          race()
        when 'creation'
          client_creation()
        end
      end
```
but it just meant more code, when I could just call the `race` and `client`creation` methods as needed and skip the `call` method.

If there is a proper way to direct these calls and using the `call` method is ideal, I'm happy to do it.

Reply:
I would break this out into two separate services, and have something higher up decide which service to call

Reply:
@jgayfer fixed.